src/service/axiosInstance.jsx

```diff
@@ -1,7 +1,28 @@
 import axios from 'axios';
+import { getSessionStorage } from '../utils/sessionStorage';
+import { UNAUTHENTICATED_ROUTES, BASE_URL } from '../constants/constants';
 
 const axiosInstance = axios.create({
-  baseURL: 'https://36b32v1d09.execute-api.sa-east-1.amazonaws.com/',
+  baseURL: BASE_URL,
+});
+
+axiosInstance.interceptors.request.use((req) => {
+  const token = getSessionStorage('token');
+
+  const isUnauthenticatedRoute = UNAUTHENTICATED_ROUTES.includes(req.url);
+
+  if (isUnauthenticatedRoute) {
+    return req;
+  }
+
+  if (token) {
+    req.headers = {
+      Authorization: `Bearer ${token}`,
+    };
+    return req;
+  }
+
+  return console.log('token não informado no sessionStorage');
```

Comment:
Na minha opinião, a melhor abordagem nessa situação (o token deveria estar configurado mas não foi por alguma razão) é levantar uma exceção.

```suggestion
  throw new Error('token não informado no sessionStorage')
```

Mensagens em inglês de preferência

```suggestion
  throw new Error('missing token in sessionStorage')
```

Reply:
:heavy_check_mark: